omnipaxos_core/src/util.rs

```diff
@@ -174,6 +178,15 @@ where
             .collect()
     }
 
+    pub fn get_unpromised_peers(&self) -> Vec<u64> {
+        self.decided_indexes
+            .iter()
+            .enumerate()
+            .filter(|(pid, x)| x.is_none() && *pid != Self::pid_to_idx(self.n_leader.pid))
+            .map(|(idx, _)| (idx + 1) as u64)
+            .collect()
+    }
```

Comment:
```suggestion
    pub fn get_unpromised_peers(&self) -> Vec<NodeId> {
        self.decided_indexes
            .iter()
            .enumerate()
            .filter(|(pid, x)| x.is_none() && *pid != Self::pid_to_idx(self.n_leader.pid))
            .map(|(idx, _)| (idx + 1) as NodeId)
            .collect()
    }
```

Reply:
I had made the same mistake for `get_promised_followers()`. Could please also update it there?